src/graphql/search/map/normalize.ts

```diff
@@ -1,116 +0,0 @@
-import queryString, { ParsedUrlQueryInput } from 'querystring'
```

Comment:
I'm going to see if I can restore this logic and align it with my implementation as to not break things by accident.

Reply:
Cool, but this will involve testing the complete module to add and remove maplayers in the atlas project. Beware of the function `composeId`, that alters the ID for maplayers when used as legend item for another maplayer. And go through all the comments that explain why certain fields are used